gandi/cli/core/conf.py

```diff
@@ -187,6 +203,14 @@ def configure(cls, global_, key, val):
         cls.save(os.path.expanduser(conf_file), config)
 
     @classmethod
+    def list(cls, global_):
+        """ Return configuration file content. """
+        scope = 'global' if global_ else 'local'
+        config = cls._conffiles.get(scope, {})
+        if config:
```

Comment:
You should always return the dirct even if it's empty, this give the possibility to call the function in a for loop without any error.  
so `return cls._conffiles.get(scope, {})`